src/components/Header/subcomponents/NavLinks.js

```diff
@@ -4,6 +4,10 @@ import { NavLink, Link } from "react-router-dom";
 import { UserConsumer } from "../../../utils/UserContext";
 
 export default function NavLinks({ toggleFormIsOpen, formIsOpen }) {
+  function logoutClicked(setUserLoggedIn) {
```

Comment:
This function is a duplicate with UsernameDropdown. How about make a common service file for logging in/out?